monai/utils/type_conversion.py

```diff
@@ -97,26 +97,29 @@ def convert_to_tensor(data, wrap_sequence: bool = False):
 
     """
     if isinstance(data, torch.Tensor):
-        return data.contiguous()
+        if device is None:
```

Comment:
I think maybe no need to check `None`, `to()` can support None:
https://github.com/pytorch/ignite/blob/master/ignite/utils.py#L28
@wyli What do you think?

Thanks.

Reply:
Thanks, fixed.